src/System.Device.Gpio/System/Device/Gpio/ExceptionHelper.cs

```diff
@@ -76,6 +76,9 @@ private static string GetResourceString(ExceptionResource resource, int errorCod
                 case ExceptionResource.LibGpiodNotInstalled:
                     message = $"Libgpiod driver not installed. More information on: https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/about/";
                     break;
+                case ExceptionResource.GpioChipControllerAlreadySet:
+                    message = $"GpioController Default gpiochip already set.";
```

Comment:
Seems you had idea that first loaded chip shouldn't be changed, by setting and loading the chip on the constructor we wouldn't need to worry about further chip changes and wouldn't need this error